crates/core/transaction/src/gas.rs

```diff
@@ -216,21 +227,71 @@ fn delegator_vote_gas_cost() -> Gas {
 
 fn position_withdraw_gas_cost() -> Gas {
     Gas {
-        // position ID = 64 + 64 bytes
-        // balance commitment = 64 bytes
-        // uint64 = 8 bytes
-        block_space: 64 + 64 + 64 + 8,
+        // PositionId `position_id`                                        = 32 bytes
+        // penumbra.core.asset.v1.BalanceCommitment `reserves_commitment`  = 32 bytes
+        // uint64 `sequence`                                               = 8 bytes
+
+        // The block space measured as the byte length of the encoded action.
+        block_space: 72, // 72 bytes
         // The compact block space cost is based on the byte size of the data the [`Action`] adds
         // to the compact block.
         // For a PositionWithdraw the compact block is not modified.
-        compact_block_space: 0u64,
+        compact_block_space: 0,
         // Does not include a zk-SNARK proof, so there's no verification cost.
         verification: 0,
         // Execution cost is currently hardcoded at 10 for all Action variants.
         execution: 10,
     }
 }
 
+fn dutch_auction_schedule_gas_cost(dutch_action_schedule: &ActionDutchAuctionSchedule) -> Gas {
+    Gas {
+        // penumbra.core.asset.v1.Value `input` = 48 bytes
+        // penumbra.core.asset.v1.AssetId `output_id` = 32 bytes
+        // penumbra.core.num.v1.Amount `max_output` = 16 bytes
+        // penumbra.core.num.v1.Amount `min_output` = 16 bytes
+        // uint64 `start_height` = 8 bytes
+        // uint64 `end_height` = 8 bytes
+        // uint64 `step_count` = 8 bytes
+        // bytes `nonce` = 32 bytes
+
+        // TODO: Multidimentional fees will enable gas fees to dynamically be based on both `block_space`
```

Comment:
I would remove this comment, it doesn't seem (?) directly relevant to the schedule action, and I think it's technically off because what we're missing is the dynamic price  adjustments based on usage/utilization targets.

Reply:
forgot to remove this!